supportutilsPluginSUSEPublicCloud/supportconfig/resources/rpmlist_EC2.rc

```diff
@@ -5,4 +5,11 @@ python-ec2metadata
 python-ec2metadata
 python-ec2publishimg
 python-ec2uploadimg
+python3-ec2deprecateimg
+python3-ec2metadata
+python3-ec2metadata
```

Comment:
Why is `python3-ec2metadata` listed twice? Looking at `rpmlist_EC2.rc` directly, this might be a copy and paste error, as there already is `python-ec2metadata` listed twice.